File: Python/10-Chandu's-Gangsters.py
arr = list(map(int,input().split())) # Taking Input
summ = sum(arr) # Calculating Sum of Array
target = summ - 100 # Calculating the target which makes the sum of array greater than 100
nums = set() 
for i in arr: # In this loop we're finding and removing those two elements whose sum is equal to the target.
  temp = target - i 
  if temp in nums:
    arr.remove(i)
    arr.remove(temp)
    break
  nums.add(i)
print(*arr) # Printing the result

File: C++/10-Chandu's-Gangsters.cpp
#include <iostream>
using namespace std;
#include<stdio.h>
int main(){
    int a[9],i,target,sum=0;
    for(i=0;i<9;i++){
        cin>>a[i]; // Taking Input
        sum += a[i]; // Calculating Sum of Array
    }
    target = sum - 100; // Calculating the target which makes the sum of array greater than 100
    int extraA,extraB; // Variables to store two numbers which makes the sum of array greater than 100
    for(i=0;i<8;i++){
        for(int j=i+1;j<9;j++){
            if(a[i] + a[j] == target){
                extraA = a[i];
                extraB = a[j];
                break;
            }
        }
    }
    for(i=0;i<9;i++){
        if(a[i]!=extraA && a[i]!=extraB)
            cout<<a[i]<<" "; // Printing the result
    }
}
